public/admin/js/editor.js

```diff
@@ -0,0 +1,3 @@
+$(function () {
+    CKEDITOR.replace('editor1');
```

Comment:
`editor1` is a not good name. And I think we shouldn't use CKEDITOR, it's too heavy. Use bootstrap wysiwyg instead